dialogueMaker/dialogueMaker.h

```diff
@@ -40,6 +43,7 @@ class Scene {
 };
 
 class Game {
+  static player Player;
```

Comment:
```suggestion
  static inline player Player;
```
To solve above issue.